app/src/main/java/io/bloco/circles/ui/BaseViewModel.kt

```diff
@@ -1,10 +1,11 @@
 package io.bloco.circles.ui
 
+import androidx.lifecycle.LifecycleObserver
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.plus
 
-abstract class BaseViewModel : ViewModel() {
+abstract class BaseViewModel : ViewModel(), LifecycleObserver {
```

Comment:
I couldn't find the reason you would need to observe the lifecycle in a ViewModel.